nexus/tests/integration_tests/subnet_allocation.rs

```diff
@@ -30,7 +29,20 @@ async fn create_instance_expect_failure(
     client: &ClientTestContext,
     url_instances: &String,
     name: &str,
+    subnet_name: &str,
 ) -> HttpErrorResponseBody {
+    let network_interfaces =
+        params::InstanceNetworkInterfaceAttachment::Create(vec![
+            params::NetworkInterfaceCreate {
+                identity: IdentityMetadataCreateParams {
+                    name: name.parse().unwrap(),
```

Comment:
Nit: Could we comment "we're using the instance name as the NIC name too", just to make it clear this is intentional?

Reply:
Yessir